testing/BUILD.bazel

```diff
@@ -18,7 +18,7 @@ load("//testing:testing_test.bzl", "testing_test")
 kt_android_library(
      name = "testing",
      custom_package = "org.oppia.testing",
-     srcs = glob(["src/main/java/**/*.kt"]),
+     srcs = glob(["src/main/java/org/oppia/testing/*.kt"]),
      manifest = "src/main/AndroidManifest.xml",
```

Comment:
Please don't depend on app module's manifest.

Reply:
I believe this is pointing to the testing module manifest

Reply:
Good call--I misread. :)